dashboard/static/css/layout.css

```diff
@@ -17,19 +39,21 @@ body {
 }
 
 .top-bar ul {
-    background-color: #424F5A;
+    background-color: transparent;
 }
 
-main {
-    margin-top: 20px;
-    margin-bottom: 20px;
+.top-bar ul.submenu {
+    background-color: #00549E;
 }
 
-form {
-    margin: 20px 0;
+.breadcrumbs a {
+    color: #00549E;
 }
 
-footer {
-    height: 100px;
-    width: 100%;
+.button {
+    background-color: #00549E;
+}
+
+.center {
+    text-align: center;
 }
```

Comment:
If we can, we should avoid presentational class names like `center` that hardcode stylistic choices into the markup. It's the same reason `<center>` can be problematic.

[Use class with semantics in mind](https://www.w3.org/QA/Tips/goodclassnames)